src/Dashboard/Types/NodesStatistics/Statistic.php

```diff
@@ -190,8 +201,10 @@ protected function prepareChartData()
      * @param int $length
      * @return string
      */
-    private static function randomString($length = 16) {
+    private static function randomString($length = 16)
```

Comment:
Of course we started with that, but it puts in the numbers which was not something dennis wanted